app/views/categories/_form.html.erb

```diff
@@ -0,0 +1,59 @@
+<section class="vh-100" style="background-color: #eee;">
+  
+  <div class="container h-90">
+    <div class="row d-flex justify-content-center align-items-center h-100">
+      <div class="col-lg-12 col-xl-11">
+        <div class="card text-black" style="border-radius: 25px;">
+          <div class="card-body p-md-5">
+            <div class="row justify-content-center">
+              <div class="col-md-10 col-lg-6 col-xl-5 order-2 order-lg-1">
+
+
+                <%= form_with(model: category) do |form| %>
+                  <% if category.errors.any? %>
+                    <div style="color: red">
+                      <h2><%= pluralize(category.errors.count, "error") %> prohibited this category from being saved:</h2>
+
+                      <ul>
+                        <% category.errors.each do |error| %>
+                          <li><%= error.full_message %></li>
+                        <% end %>
+                      </ul>
+                    </div>
+                  <% end %>
+
+                  <div>
+                    <%= form.text_field :name, class:"form-control", placeholder: "Name" %>
+                  </div>
+
+                  <div>
+                    <%= form.text_field :icon, class:"form-control mt-3 mb-3", placeholder: "Only valid image url" %>
+                  </div>
+
+                  <div>
+                    <%= form.text_field :author_id, value: current_user.id, class:"form-control", placeholder: "Name", hidden: true %>
+                  </div>
+
+                  <div>
+                    <%= form.submit "Add Category", class: "btn btn-primary form-control" %>
+                  </div>
```

Comment:
- [ ] [REQUIRED] You have done a great job @brytebee  however, whenever I try to add a new category locally, the icon of the user does not get displayed. Please fix this as the name and the icon of the user should be displayed.
![Screenshot from 2022-05-28 01-01-34](https://user-images.githubusercontent.com/58692590/170801570-08027a54-ee37-42e7-9445-762e6913e8a7.png)